code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -382,102 +415,79 @@
-		"MAX_TEMP" = T0C + 70 //Pretty warm, explicitly not fire temps. Time bombs are cool, but not that cool. If it makes you feel any better it's close
-	)
-
-/datum/gas_reaction/nitrium_decomposition/react(datum/gas_mixture/air)
-	var/list/cached_gases = air.gases
-	var/temperature = air.temperature
-	var/old_heat_capacity = air.heat_capacity()
-
-	//This reaction is agressively slow. like, a tenth of a mole per fire slow. Keep that in mind
-	var/heat_efficency = min(temperature / (FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 8), cached_gases[/datum/gas/nitrium][MOLES])
-	var/energy_produced = heat_efficency * NITRIUM_DECOMPOSITION_ENERGY
-
-	if ((cached_gases[/datum/gas/nitrium][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
-		return NO_REACTION
-
-	air.assert_gases(/datum/gas/nitrogen, /datum/gas/hydrogen)
-	cached_gases[/datum/gas/nitrium][MOLES] -= heat_efficency
-	cached_gases[/datum/gas/hydrogen][MOLES] += heat_efficency
-	cached_gases[/datum/gas/nitrogen][MOLES] += heat_efficency
+	var/energy_used = heat_efficency * N2O_FORMATION_ENERGY
+	var/new_heat_capacity = air.heat_capacity()
+	if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+		air.temperature = max(((air.temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB) // The air cools down when reacting.
+	return REACTING
 
-	if(energy_produced> 0)
-		var/new_heat_capacity = air.heat_capacity()
-		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
-			air.temperature = max(((temperature * old_heat_capacity + energy_produced) / new_heat_capacity), TCMB) //the air heats up when reacting
-		return REACTING
 
-/datum/gas_reaction/nitrium_formation //The formation of nitrium. Endothermic. Requires bz.
-	priority_group = PRIORITY_FORMATION
-	name = "Nitrium formation"
-	id = "nitrium_formation"
+/**
+ * Nitrous Oxide Decomposition
+ *
+ * Decomposition of N2O.
+ * Exothermic.
+ */
+/datum/gas_reaction/nitrous_decomp
+	priority_group = PRIORITY_POST_FORMATION
+	name = "Nitrous Oxide Decomposition"
+	id = "nitrous_decomp"
 
-/datum/gas_reaction/nitrium_formation/init_reqs()
+/datum/gas_reaction/nitrous_decomp/init_reqs()
 	requirements = list(
-		/datum/gas/tritium = 20,
-		/datum/gas/nitrogen = 10,
-		/datum/gas/bz = 5,
-		"MIN_TEMP" = 1500
+		/datum/gas/nitrous_oxide = MINIMUM_MOLE_COUNT * 2,
+		"MIN_TEMP" = N2O_DECOMPOSITION_MIN_TEMPERATURE,
+		"MAX_TEMP" = N2O_DECOMPOSITION_MAX_TEMPERATURE,
 	)
 
-/datum/gas_reaction/nitrium_formation/react(datum/gas_mixture/air)
-	var/list/cached_gases = air.gases
+/datum/gas_reaction/nitrous_decomp/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases //this speeds things up because accessing datum vars is slow
 	var/temperature = air.temperature
+	var/burned_fuel = (cached_gases[/datum/gas/nitrous_oxide][MOLES] / N2O_DECOMPOSITION_RATE_DIVISOR) * ((temperature - N2O_DECOMPOSITION_MIN_SCALE_TEMP) * (temperature - N2O_DECOMPOSITION_MAX_SCALE_TEMP) / (N2O_DECOMPOSITION_SCALE_DIVISOR))
```

Comment:
Could you explain how 
`burned_fuel = max(0,0.00002 * (temperature - (0.00001 * (temperature**2)))) * cached_gases[/datum/gas/nitrous_oxide][MOLES]` 
became this?
Also why wrap `N2O_DECOMPOSITION_SCALE_DIVISOR` in ()'s?

Reply:
That equation made the decomp rate a parabola with roots at 0 and 100000 and which maxed out at about 50% of the N2O per tick. I moved the 50% max to the decomp rate divisor and the parabola bit into the quadratic. This just shows the roots and the maximum value a bit more explicitly. The scale divisor is the bit that makes the quadratic scaling factor max out at 1 instead of at -2.5 billion.

Reply:
I'm not certain why the scale divisor is wrapped in parens. I think I had it explicitly written out at some point and that may have been a holdover.